extensions/control-plane/api/data-management/asset-api/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/AssetApi.java

```diff
@@ -35,13 +36,14 @@ public interface AssetApi {
 
     @Operation(description = "Creates a new asset together with a data address",
             responses = {
-                    @ApiResponse(responseCode = "200", description = "Asset was created successfully"),
+                    @ApiResponse(responseCode = "200", description = "Asset was created successfully. Returns the asset Id",
+                            content = @Content(schema = @Schema(implementation = AssetId.class))),
                     @ApiResponse(responseCode = "400", description = "Request body was malformed",
                             content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiErrorDetail.class)))),
                     @ApiResponse(responseCode = "409", description = "Could not create asset, because an asset with that ID already exists",
                             content = @Content(array = @ArraySchema(schema = @Schema(implementation = ApiErrorDetail.class)))) }
     )
-    void createAsset(@Valid AssetEntryDto assetEntryDto);
+    AssetId createAsset(@Valid AssetEntryDto assetEntryDto);
```

Comment:
Please do not use these unnecessary wrappers. Just return a `String`.

Reply:
yes, I agree. I just tried to implement the same thing as contractnegotiation-api , 
org.eclipse.dataspaceconnector.api.datamanagement.contractnegotiation.model.NegotiationId

Reply:
looking at the other endpoints responses, using String will breaks the consistency of the api responses, since all the other  endpoints either return nothing or a json. Furthermore, as I mentioned contractnegotiation-api and transferprocess-api are already implemented with the same approach(Using a wrapper for Id). 
if this argument is convincing, then I can use the subclasses of BaseResponseDto not instead of wrappers. However my question is, if I should use the ones which are already implemented such as AssetResponseDto or ContractDefinitionResponseDto, which returns the whole object or a new one to return only the Id?